app/src/main/java/de/tum/in/tumcampusapp/component/ui/overview/MainFragment.kt

```diff
@@ -95,6 +100,19 @@ class MainFragment : BaseFragment<Unit>(
         viewModel.cards.observe(viewLifecycleOwner) {
             it?.let { onNewCardsAvailable(it) }
         }
+
+        // Triggers a Google Play store review if the user has experienced some key features of the app
+        // The earliest possible re-trigger of a review prompt occurs after half a year
+        val dateFormatter = SimpleDateFormat("MM-yyyy", Locale.GERMANY)
+        val lastReviewDate = dateFormatter.parse(Utils.getSetting(requireContext(), Const.LAST_REVIEW_PROMPT, "01-01-1970"))
+        val halfYearInSeconds = 6*30*24*60*60
```

Comment:
This could be a constant (in a companion object)

Reply:
These constants are no longer present in the code due to the other suggestion, that you made (except for lastReviewDate). Should I move that to a companion object still?